temporian/implementation/numpy/data/plotter.py

```diff
@@ -0,0 +1,156 @@
+from typing import NamedTuple, Optional, Union, List, Any
+from temporian.implementation.numpy.data.event import NumpyEvent
+
+import numpy as np
+
+DEFAULT_BACKEND = "matplotlib"
+
+
+class Options(NamedTuple):
+    """Options for plotting."""
+
+    backend: str
+    height_per_plot_px: int
+    width_px: int
+    max_points: Optional[int]
+
+
+def plot(
+    event: Union[List[NumpyEvent], NumpyEvent],
+    index: Union[tuple, Any] = (),
+    backend: str = DEFAULT_BACKEND,
+    width_px: int = 1024,
+    height_per_plot_px: int = 150,
+    max_points: Optional[int] = None,
+):
+    """Plots an event.
+
+    Args:
+        index: The index of the event to plot. Use 'event.index()' for the
+            list of available indices.
+        backend: Plotting library to use.
+        width_px: Width of the figure in pixel.
+        height_per_plot_px: Height of each sub-plot (one per feature) in pixel.
+        max_points: Maximum number of points to plot.
+    """
+
+    if not isinstance(index, tuple):
+        index = (index,)
+
+    options = Options(
+        backend=backend,
+        width_px=width_px,
+        height_per_plot_px=height_per_plot_px,
+        max_points=max_points,
+    )
+
+    if backend not in BACKENDS:
+        raise ValueError(
+            f"Unknown plotting backend {backend}. Available "
+            "backends: {BACKENDS}"
+        )
+
+    if isinstance(event, list):
+        events = event
+    else:
+        events = [event]
+
+    return BACKENDS[backend](events=events, index=index, options=options)
+
+
+def _plot_matplotlib(events: List[NumpyEvent], index: tuple, options: Options):
+    import matplotlib.pyplot as plt
+    from matplotlib.cm import get_cmap
+
+    colors = get_cmap("tab10").colors
+
+    px = 1 / plt.rcParams["figure.dpi"]
+
+    num_plots = 0
+    for event in events:
+        if index not in event.data:
+            raise ValueError(
+                f"Index '{index}' does not exist in event. Check the available"
+                " indexes with 'event.index()' and provide one of those index"

```

Comment:
Can't we list `event.index()` in the error for better clarity?

Maybe say something like:
```python
f"Index '{index}' does not exist in the event. Possible indexes: {event.index()}"
```

Reply:
Good idea.
*Done